content/en/docs/tasks/security/ca-config/plugin-ca-cert/index.md

```diff
@@ -38,66 +35,53 @@ These files are ready to use in the `samples/certs/` directory.
   You must use these specific secret and file names, or reconfigure Citadel when you deploy it.
   {{< /tip >}}
 
-The following steps enable plugging in the certificates and key into Citadel:
+The following steps plug in the certificates and key into a Kubernetes secret,
+which will be read by Citadel:
 
-1.  Create a secret `cacert` including all the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`:
+1.  Create a secret `cacerts` including all the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`:
 
     {{< text bash >}}
+    $ kubectl create namespace istio-system
     $ kubectl create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem \
         --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem \
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1.  Redeploy Citadel with `global.mtls.enabled` set to `true` and `security.selfSigned` to `false`.
+1.  Deploy Citadel with `global.mtls.enabled` set to `true` and the `demo` profile.
     Citadel will read certificates and key from the secret-mount files.
 
     {{< text bash >}}
-    $ istioctl manifest apply --set values.global.mtls.enabled=true --set values.security.selfSigned=false
-    {{< /text >}}
-
-1.  To make sure the workloads obtain the new certificates promptly,
-    delete the secrets generated by Citadel (named as `istio.\*`).
-    In this example, `istio.default`. Citadel will issue new certificates for the workloads.
-
-    {{< text bash >}}
-    $ kubectl delete secret istio.default
+    $ istioctl manifest apply --set profile=demo --set values.global.mtls.enabled=true
```

Comment:
Same as other, remove `--set values.global.mtls.enabled=true`

Reply:
This guide requires mTLS (e.g., mTLS between sleep and httpbin sample services). If removing "--set values.global.mtls.enabled=true", how to make sure that mTLS is enforced?